MmSupervisorPkg/Core/Mem/SmmCpuMemoryManagement.c

```diff
@@ -2150,3 +2308,185 @@ SetUnblockRegionAttribute (
   DEBUG ((DEBUG_INFO, "%a - Exit - %r\n", __FUNCTION__, Status));
   return Status;
 }
+
+/**
+  Prevent the memory pages used for SMM page table from been overwritten.

```

Comment:
Prevent the memory pages used for SMM page table from `being` overwritten.

Reply:
Thanks, updated.